### ISSUE TITLE
How to import excel when heading row is different?

### ISSUE THREAD

**Issue by yaofong** [rap2hpoutre/fast-excel#86]
How to import excel when my table doesn't start from A1.

For example, my row started at B4, and the value start from B5.

**Comment by rap2hpoutre** [comment]
There is no built-in way to do this. Still you could consider building your own rules by skipping header and using your own callback function:

```php
$users = (new FastExcel)->withoutHeaders()->import('file.xlsx', function ($line) {
    // Skip if empty or first line (i.e. header), ... 
    // ... return the object otherwise (without the first columns)
});

```

**Comment by yaofong** [comment]
Thanks @rap2hpoutre for replying to my question.

However, when I add `->withoutHeaders()`, it doesn't runs the callback.

**Comment by realtebo** [comment]
Same issue, when using `->withoutHeaders()`, I can't use the callback.

**Comment by rap2hpoutre** [comment]
@realtebo @yaofong It's now fixed thanks to you.

**It's available in v1.2.2**

I also added a test for this issue: https://github.com/rap2hpoutre/fast-excel/blob/1731095b20787ecb53b7a9d48032f452454ddd02/tests/IssuesTest.php#L150-L157

@yaofong Sorry for late answer :/

**Comment by RayhanYulanda** [comment]
> There is no built-in way to do this. Still you could consider building your own rules by skipping header and using your own callback function:
> 
> ```
> $users = (new FastExcel)->withoutHeaders()->import('file.xlsx', function ($line) {
>     // Skip if empty or first line (i.e. header), ... 
>     // ... return the object otherwise (without the first columns)
> });
> ```

Please write it in readme. Thanks a lot
